src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of 'Random' every time the method is called can be inefficient. Consider creating a single instance of 'Random' as a class-level field and reusing it. This will ensure that you're not unnecessarily creating new objects each time, which can be particularly important in a method that might be called frequently.